mapflow/mapflow.py

```diff
@@ -670,6 +687,16 @@ def on_project_change(self, currentIndex):
         # Manually toggle function to avoid race condition
         self.calculate_aoi_area_use_image_extent(self.dlg.useImageExtentAsAoi.isChecked())
         self.setup_processings_table()
+        if not self.user_role.can_delete_rename_project:
```

Comment:
Also, move it to the mainDialog.

Like function mainDialog.setup_for_user_role(userRole) and call it any where it is needed, of more specific, like setup_processing_start/ setup_processing_rating/ setup_wd_options